templates/layout.html

```diff
@@ -66,7 +66,7 @@
                       ['/contributing-code/projects', 'Project List'],
                       ['/contributing-code/pr-guidelines', 'Pull Request Guidelines'],
                       ['/contributing-code/github-repo-guidelines', 'GitHub Repo Guidelines'],
-                      ['/contributing-code/cc-search', 'CC Search'],
```

Comment:
You're removing the existing page here – that is still an important page as well. You can rename it to "Contributing to CC Search".

I recommend making a new top level item for CC Search (not under Contribute) for this page for now.